spec/models/slide_spec.rb

```diff
@@ -2,45 +2,86 @@
 require 'support/model_with_a_position_attribute'
 
 describe Slide do
-  it { is_expected.to validate_presence_of :key }
-  it_behaves_like 'a model with a position attribute'
-
-  describe '#url_components' do
-    context 'when the slide has a sidebar item and tab' do
-      let(:slide) { FactoryBot.create(:slide) }
-
-      it 'returns the tab key first' do
-        expect(slide.url_components[0]).to eq(slide.sidebar_item.tab.key)
-      end
-
-      it 'returns the sidebar item key second' do
-        expect(slide.url_components[1]).to eq(slide.sidebar_item.key)
-      end
-
-      it 'returns the slide name last' do
-        expect(slide.url_components[2]).to eq(slide.short_name)
-      end
-    end
-
-    context 'when the slide has no sidebar item' do
-      let(:slide) { FactoryBot.create(:slide, sidebar_item: nil) }
-
-      it 'returns an empty array' do
-        expect(slide.url_components).to eq([])
-      end
-    end
-
-    context 'when the sidebar item has no tab' do
-      let(:slide) do
-        FactoryBot.create(
-          :slide,
-          sidebar_item: FactoryBot.create(:sidebar_item, tab: nil)
-        )
-      end
-
-      it 'returns an empty array' do
-        expect(slide.url_components).to eq([])
-      end
-    end
+  describe '.' do
+    subject { Slide }
+    it { is_expected.to respond_to :ordered }
+    it { is_expected.to respond_to :controller }
+    it { is_expected.to respond_to :visible }
   end
+
+  describe '.new' do
```

Comment:
FYI, the master branch [has (hopefully) fixed the Hound config](https://github.com/quintel/etmodel/commit/ed74fd81195571d8f154ff5c70c1598984b15eb9) so that the correct Rubocop settings are used; you might wanna rebase to put an end to these false positives. This cop in particular [is disabled](https://github.com/quintel/etmodel/blob/7830ff9fe12b3f86925ec80ad3637dec75d4d0db/.rubocop.yml#L123-L126), since limiting block length in specs seems silly.